fail2ban/tests/utils.py

```diff
@@ -368,6 +448,38 @@ def assertNotIn(self, a, b, msg=None):
 
 class LogCaptureTestCase(unittest.TestCase):
 
+	class _MemHandler(logging.StreamHandler):
+		def __init__(self):
+			self._recs = list()
+			self._strm = StringIO()
+			logging.StreamHandler.__init__(self, self._strm)
+			# 
```

Comment:
sorry to be a stylistic pain but why to have a `#` here?  we have them nowhere else (and I hope will not). so why?